runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectPoint.java

```diff
@@ -44,7 +44,8 @@ class ObjectPoint
 	private int regionX;
 	private int regionY;
 	private int z;
-	private Color color;
+	private Color borderColor;
```

Comment:
This object is serialized and saved in the config, so this change will cause all marked objects to reset their color. Either don't rename it or use `@SerializedName`